src/cryptonote_basic/hardfork.h

```diff
@@ -213,9 +213,9 @@ class HardFork
 	uint8_t get_current_version_num() const;
 
 	/**
-     * @brief returns the earliest block a given version may activate
+     * @brief returns the earliest block a known version may activate
```

Comment:
could you please add `@return if unknown version than `largest 64bit unsigned integer` else activation block height`